src/js/utils/SchemaUtil.js

```diff
@@ -61,7 +61,7 @@ function schemaToFieldDefinition(fieldName, fieldProps, formParent, isRequired,
     definition.checked = value;
   }
 
-  if (fieldProps.fieldType === 'number') {
+  if (fieldProps.type === 'number') {
```

Comment:
Thanks guys! I searched the codebase and it looks like all `fieldType` keys are meant to be there (ie. they belong to an actual definition, not schema). Good call!